upstream.wrap

```diff
@@ -0,0 +1,8 @@
+[wrap-file]
+directory=gc-7.6.8
+
+source_url=https://github.com/ivmai/bdwgc/releases/download/v7.6.8/gc-7.6.8.tar.gz
+source_filename=gc-7.6.8.tar.gz
+source_hash=040ac5cdbf1bebc7c8cd4928996bbae0c54497c151ea5639838fa0128102e258
+
+X-wrapdev-name=gc
```

Comment:
What is `X-wrapdev-name`?

Reply:
It's the name of the project and dependency (which in this case is wrong) so that wrapdevtools can generate a project to test the wrap. I can remove it for now, and wait on that feature for some of xclaesse's wrap commits.